src/components/ContinuationIn/ManualBusinessInfo.vue

```diff
@@ -45,7 +45,7 @@
             class="home-jurisdiction"
             :showUsaJurisdictions="true"
             label="Jurisdiction"
-            :initialValue="jurisdictionInitialVal"
+            :initialValue="business.homeJurisdiction"
```

Comment:
In a previous ticket, I updated this shared Jurisdiction component to properly initialize, so passing in this object is all that's needed here.